include/glow/Importer/ProtobufLoader.h

```diff
@@ -181,8 +189,62 @@ class ProtobufLoader {
   /// \returns the Placeholder for the external output with \p name.
   /// \pre outputVarsByName_.find(name) != outputVarsByName_.end()
   llvm::Expected<Placeholder *> getOutputByName(llvm::StringRef name) const;
+
+  /// \returns True if the operator with name \p typeName having input node
+  /// list as \p inputs is constant foldable.
+  bool isConstantFoldable(llvm::SmallVector<NodeValue, 4> &inputs,
```

Comment:
It is better to use `llvm::SmallVectorImpl<NodeValue> &` or even better `llvm::ArrayRef<NodeValue>` in such cases to support any size of the passed vector.